linux_os/guide/services/base/service_irqbalance_enabled/rule.yml

```diff
@@ -14,14 +14,15 @@ rationale: |-
     In an environment with multiple processors (now common), the irqbalance service
     provides potential speedups for handling interrupt requests.
 
-severity: unknown
+severity: low
 
 identifiers:
     cce@rhel6: 26990-2
     cce@rhel7: 80257-9
+    cce@rhel8: 82454-0
```

Comment:
@shawndwells please remove these CCEs and place back in the cce pool. irqbalance is not provided for RHEL8.

Reply:
Done